app/src/androidTest/java/net/skyscanner/backpack/compose/cardwrapper/BpkCardWrapperTest.kt

```diff
@@ -39,102 +43,88 @@ import org.junit.Test
 class BpkCardWrapperTest : BpkSnapshotTest() {
 
     @Test
-    fun wrapperWidthSmallerThanMinimum() {
+    fun default() {
```

Comment:
mainly simplified the tests here to reuse things and also removed some use cases that weren't actually relevant for the card wrapper (and added some that were)